recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+fortran_compiler:              # [unix or win64]
+  - gfortran                   # [(linux64 or osx)]
+  - gfortran                   # [aarch64 or ppc64le or armv7l]
+fortran_compiler_version:      # [unix or win64]
+  - 7                          # [linux64 or osx or aarch64]
+  - 8                          # [ppc64le or armv7l]
```

Comment:
You should remove this file after https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/286

Reply:
Sorry I'm confused, are we removing the `conda_build_config.yaml` file? Could I have more context?

Reply:
We are not building with gfortran 4 after that PR is merged, so there's nothing to skip.